templates/prometheus.yaml

```diff
@@ -90,6 +94,25 @@ spec:
           thanos:
             baseImage: thanosio/thanos
             version: v0.4.0
+          externalLabels:
+            cluster: $(CLUSTER_ID)
```

Comment:
Why do we need this pod sleeping forever?

Reply:
Without the sleep, the container starts up then immediately exits after completing all the tasks, and causes a `CrashLoopBackOff`. IIUC once we update the prometheus-operator helm chart to a higher version, we can move this container into `initContainers` (not supported in current version) and avoid the sleep

Reply:
https://jira.mesosphere.com/browse/DCOS-60304 for prometheus TODO once we upgrade to a version that supports `initContainers`